src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -0,0 +1,209 @@
+        it('should find faqs by courseId', async () => {
+            const category = {
+                color: '#6ae8ac',
+                category: 'category1',
+            } as FAQCategory;
+            const returnedFromService = [{ ...elemDefault, categories: [JSON.stringify(category)] }];
+            const expected = [{ ...elemDefault, categories: [new FAQCategory('category1', '#6ae8ac')] }];
+            const courseId = 1;
+            service
+                .findAllByCourseId(courseId)
+                .pipe(take(1))
+                .subscribe((resp) => (expectedResult = resp));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)